src/mumble/Log.cpp

```diff
@@ -64,6 +66,8 @@ LogConfig::LogConfig(Settings &st) : ConfigWidget(st) {
 	allMessagesItem->setToolTip(ColNotification, QObject::tr("Toggle pop-up notifications for all events"));
 	allMessagesItem->setCheckState(ColHighlight, Qt::Unchecked);
 	allMessagesItem->setToolTip(ColHighlight, QObject::tr("Toggle window highlight (if not active) for all events"));
+	allMessagesItem->setCheckState(ColMessageLimit, Qt::Unchecked);
+	allMessagesItem->setToolTip(ColMessageLimit, QObject::tr("Click here to toggle message limiting for all events - If using this option be sure to change the user limit below."));
```

Comment:
```suggestion
	allMessagesItem->setToolTip(ColMessageLimit, tr("Click here to toggle message limiting for all events - If using this option be sure to change the user limit below."));
```
We recently figure out that one should not use `QObject::` to access this function as Qt performas some behind-the-scene magic that involves the scope at which this function is invoked.